src/game/server/gamemodes/ictf.cpp

```diff
@@ -23,4 +24,11 @@ void CGameControllerICTF::OnCharacterSpawn(class CCharacter *pChr)
 
 	// give default weapons
 	pChr->GiveWeapon(WEAPON_LASER, false, g_Config.m_SvGrenadeAmmoRegen ? g_Config.m_SvGrenadeAmmoRegenNum : -1);
+}
+
+void CGameControllerICTF::OnPlayerConnect(CPlayer *pPlayer)
+{
+	// this is the main part of the gamemode, this function is run every tick
+	pPlayer->SetTeam(TEAM_RED, false);
```

Comment:
does this bypass sv_tournament_mode ?

Reply:
I added that to more quickly playtest, didn't think it was in this file. I'll remove it and make another pr for the proper ictf join functionality (default ictf is to autojoin the team with the least players)